### PR TITLE
feat(bivariate-color-manager): 10842 - add legend details info block 

### PR DESCRIPTION
**Pull request by propakov**
https://kontur.fibery.io/Tasks/Task/Choose-denominators-for-selected-indicators-to-make-layers-(Color-legends-section)-dn-fe-10842
Screenshot 2022-08-03 at 17 58 47

### CODE REVIEW

File: src/features/bivariate_color_manager/components/LegendWithMap/LegendWithMap.tsx [hunk 1]
```diff
@@ -70,6 +82,40 @@ export const LegendWithMap = ({
...
+  <div>
+    <div className={s.LegendDetailsLabel}>
+      {label + (mostQualityDenominator ? ` to ${mostQualityDenominator}` : '')}
+    </div>
+    <div className={s.LegendDetailsDirection}>
+      {convertDirectionsArrayToLabel(direction)}
+      <EditLine24 width={16} height={16} />
+    </div>
+  </div>
+);
```
Comment (IlyaIzr) [inline src/features/bivariate_color_manager/components/LegendWithMap/LegendWithMap.tsx]: I think this component big enough to have separate file

File: src/features/bivariate_color_manager/components/LegendWithMap/LegendWithMap.tsx [hunk 2]
```diff
@@ -70,6 +82,40 @@ export const LegendWithMap = ({
...
+  mostQualityDenominator?: string;
+};
+const LegendDetails = ({
+  label,
+  mostQualityDenominator,
+  direction,
+}: LegendDetailsProps) => (
+  <div>
+    <div className={s.LegendDetailsLabel}>
+      {label + (mostQualityDenominator ? ` to ${mostQualityDenominator}` : '')}
```
Comment (IlyaIzr) [inline src/features/bivariate_color_manager/components/LegendWithMap/LegendWithMap.tsx]: 'to' will need translation

**Review by IlyaIzr — CHANGES_REQUESTED** [review]
also `joinAndCapitalizeItems(direction)` is a bit confusing to read, even considering direction is []

**Comment by propakov** [comment]
> also `joinAndCapitalizeItems(direction)` is a bit confusing to read, even considering direction is []

I remember we've renamed it already by @Akiyamka request. 
@IlyaIzr Do you have any suggestions?

**Comment by propakov** [comment]
@IlyaIzr updated naming approach.

**Review by IlyaIzr — APPROVED** [review]

**Pull request merged**